misc/dist/html/editor.html

```diff
@@ -3,27 +3,27 @@
 	<head>
 		<meta charset="utf-8">
 		<meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1, maximum-scale=1, user-scalable=no">
-		<meta name="author" content="Godot Engine">
-		<meta name="description" content="Use the Godot Engine editor directly in your web browser, without having to install anything.">
+		<meta name="author" content="Blazium Engine">
+		<meta name="description" content="Use the Blazium Engine editor directly in your web browser, without having to install anything.">
 		<meta name="mobile-web-app-capable" content="yes">
 		<meta name="apple-mobile-web-app-capable" content="yes">
-		<meta name="application-name" content="Godot">
-		<meta name="apple-mobile-web-app-title" content="Godot">
+		<meta name="application-name" content="Blazium">
+		<meta name="apple-mobile-web-app-title" content="Blazium">
 		<meta name="theme-color" content="#202531">
 		<meta name="msapplication-navbutton-color" content="#202531">
 		<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
 		<meta name="msapplication-starturl" content="/latest">
-		<meta property="og:site_name" content="Godot Engine Web Editor">
+		<meta property="og:site_name" content="Blazium Engine Web Editor">
 		<meta property="og:url" name="twitter:url" content="https://editor.godotengine.org/releases/latest/">
```

Comment:
Do we have a replacement URL for this?

Reply:
I'm not sure, not yet i think. that's why i left them untouched :)

Reply:
I think bio is wrapping up the CI/CD :)
Once we have the nightly builds are out i think we will have a link for this?